content/proposals/proc/FEP-0001.md

```diff
@@ -0,0 +1,587 @@
+the new FEP. Creating an issue in appropriate GitHub repository with 'idea' or
+'discussion' label is usually the best way to go about this, unless a more
+specialized venue is appropriate.
+
+Vetting an idea publicly before going as far as writing a FEP is meant to save
+the potential author time. Asking the FrostFS community first if an idea is
+original helps prevent too much time being spent on something that is guaranteed
+to be rejected based on prior discussions (searching the internet does not
+always do the trick). It also helps to make sure the idea is applicable to the
+entire community and not just the author. Just because an idea sounds good to
+the author does not mean it will work for most people in most areas where
+FrostFS is used.
+
+Once the champion has asked the community as to whether an idea has any chance
+of acceptance, a draft FEP should be presented to the appropriate venue
+mentioned above. This gives the author a chance to flesh out the draft FEP to
+make properly formatted, of high quality, and to address initial concerns about
+the proposal.
+
+#### Submitting a FEP
+
+Following the above initial discussion, the workflow varies based on whether any
+of the FEP's co-authors are Committers. If one or more of the FEP's co-authors
+are Committers, they are responsible for following the process outlined below.
+Otherwise (i.e. none of the co-authors are Committers), then the FEP author(s)
+will need to find a sponsor for the FEP.
+
+Ideally, a Committer sponsor is identified, but non-Committer sponsors may also
+be selected with the approval of the Architecture Committee. Members of the
+GitHub "Committers" and "Architecture Committee" teams are pre-approved to be
+sponsors. The sponsor's job is to provide guidance to the FEP author to help
+them through the logistics of the FEP process (somewhat acting like a mentor).
+Being a sponsor does **not** disqualify that person from becoming a co-author.
+The sponsor of a FEP is recorded in the "Sponsor:" field of the header.
+
+Once the sponsor or the Committer(s) co-authoring the FEP deem the document
+ready for submission, the proposal should be submitted as a draft FEP via a
+[GitHub pull request in frostfs.info
+repository](https://github.com/TrueCloudLab/frostfs.info/pulls). The draft must
+be written in FEP style as described below, else it will fail review immediately
+(although minor errors may be corrected by the editors).
+
+The standard FEP workflow is:
+
+* You, the FEP author, fork the [frostfs.info
+  repository](https://github.com/TrueCloudLab/frostfs.info/), and create a file
+  named `content/proposals/<type>/fep-X.md` that contains your new FEP. Use "X"
+  as your draft FEP number.
+
+* In the "Type:" header field, enter "Standards Track",
+  "Informational", or "Process" as appropriate, and for the "Status:"
+  field enter "Draft". For full details, see [FEP Header Preamble]({{<ref
+  "#fep_header_preamble">}}).
```

Comment:
The link is not rendered in GH web viewer.

Reply:
It's not supposed to be. For nice view the one should `make server` and use local browser against provided URL.

Maybe later we could render site PRs to a separate URL to simplify process.